core/src/test/scala/org/apache/spark/shuffle/unsafe/UnsafeShuffleSuite.scala

```diff
@@ -41,6 +41,20 @@ class UnsafeShuffleSuite extends ShuffleSuite with BeforeAndAfterAll {
     conf.set("spark.shuffle.memoryFraction", "0.5")
   }
 
+  override def multipleAttemptConfs: Seq[(String,SparkConf)] = {
+    val kryoAndNoSpillMove = conf.clone()
+      .set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
+      .set("spark.shuffle.unsafe.testing.allowSpillMove", "false")
+    val noCompression = kryoAndNoSpillMove.clone()
+      .set("spark.shuffle.compress", "false")
+      .set("spark.file.transferTo","false")
+    Seq(
+      "slow merge path" -> kryoAndNoSpillMove,
+      "filestream based fast merge" -> noCompression
+    )
```

Comment:
Did it fail when running the test locally?  Which Linux kernel version are you running?